src/v2/Apps/Order/Components/PaymentMethodSummaryItem.tsx

```diff
@@ -1,36 +1,40 @@
-import { CreditCardSummaryItem_order } from "v2/__generated__/CreditCardSummaryItem_order.graphql"
+import { PaymentMethodSummaryItem_order } from "v2/__generated__/PaymentMethodSummaryItem_order.graphql"
 import {
   StepSummaryItem,
   StepSummaryItemProps,
 } from "v2/Components/StepSummaryItem"
 import { createFragmentContainer, graphql } from "react-relay"
 import { CreditCardDetails } from "./CreditCardDetails"
+import { BankDebitDetails } from "./BankDebitDetails"
 
-const CreditCardSummaryItem = ({
+const PaymentMethodSummaryItem = ({
   order: { creditCard },
   textColor = "black100",
   ...others
 }: {
-  order: CreditCardSummaryItem_order
+  order: PaymentMethodSummaryItem_order
   textColor?: string
 } & StepSummaryItemProps) => {
   const cardInfoWithTextColor = {
     ...creditCard!,
     ...{ textColor: textColor },
   }
 
+  // TODO: check paymentMethod value from CommerceOrder to conditioanlly
+  // render details.
   return (
     <StepSummaryItem {...others}>
-      <CreditCardDetails {...cardInfoWithTextColor} />
+      {creditCard && <CreditCardDetails {...cardInfoWithTextColor} />}
+      {!creditCard && <BankDebitDetails {...cardInfoWithTextColor} />}
     </StepSummaryItem>
```

Comment:
**suggestion**: this makes sense. but, to be completely safe in case we merge this PR soon, I'd suggest to simply render `CreditCardDetails` for now as it is on main, and incorporate `BankDebitDetails` when we get the paymentMethod from the API.

Reply:
I think having it behind the ff is a good idea! 

for the merge, I'd say let's go with whatever is less complicated and time-consuming. I don't think this blocks anything, so merging when complete is totally ok too in my opinion.